mapit/views/areas.py

```diff
@@ -173,6 +174,11 @@ def area(request, area_id, format=''):
     return output_json(area.as_dict(names))
 
 
+# HACK: Some geos are too large to store in memcached.
+# PyLibMC throws an exception in this case and the request fails.
+# Setting max-age to 0 to prevent Django trying to cache in memcached.
+# Setting s-maxage so any 'upstream' caching can still be attempted.
+@cache_control(max_age=0, s_maxage=2419200)
```

Comment:
`output_polygon` in shortcuts.py sets max-age directly there - easier to change the line there than in this file in two places?